assets/src/components/form/select/model-selects/event.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Internal imports
+ */
+import ModelSelect from '../model-select';
+import * as eventModel from '../../../../data/model/event';
+
+/**
+ * External imports
+ */
+import { __ } from '@eventespresso/i18n';
+import { Component } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Select Component for the Event Model.
+ */
+export default class EventSelect extends Component {
+	state = {
+		modelName: 'event',
+	};
+
+	static defaultProps = {
+		selectConfiguration: {
+			loadingMessage: () => __( 'Retrieving Events.', 'event_espresso' ),
+			noOptionsMessage: () => __(
+				'There are no events available to select from.',
```

Comment:
maybe "No Events" instead ? This is just too long for some use cases and should be simplified.

Reply:
Am doing.